Src/Private/Get-AbrHRZRolePrivilege.ps1

```diff
@@ -56,6 +56,37 @@ function Get-AbrHRZRolePrivilege {
                         }
                         $OutObj | Sort-Object -Property 'Name' | Table @TableParams
                     }
+                    
+                    if ($InfoLevel.Settings.Administrators.RolePrivileges -ge 2) {   
+                        section -Style Heading4 "Role Privileges Details" {
+                            
+                            Paragraph "The following section details the Role Privilege details for information for $($HVEnvironment.toUpper()) server."
+                            BlankLine
+                            $OutObj = @()
+                            foreach ($Role in $Roles) {
+                                Write-PscriboMessage "Discovered Role Provilege Detailed Information for $($HVEnvironment.toUpper()) server."
+                                $inObj = [ordered] @{
+                                    'Name' = $Role.base.Name
+                                    'Description' = [string]::join("`n", $($Role.base.Privileges))
+                                }
+
+                                $OutObj += [pscustomobject](ConvertTo-HashToYN $inObj)
+                            }
+
+                            $TableParams = @{
+                                Name = "Role Privileges Details - $($Role.base.Name)"
```

Comment:
I believe $($HVEnvironment.toUpper()) is the variable to use here.

![image](https://github.com/AsBuiltReport/AsBuiltReport.VMware.Horizon/assets/1002783/daed7069-e444-4f8d-9800-b1f3f383be5a)